css/components/hero.css

```diff
@@ -0,0 +1,67 @@
+.hero__container {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  align-items: center;
+
+}
+
+@media screen and (min-width: 1024px) {
+  .hero__container {
+    flex-direction: row;
+  }
+}
+
+.hero__title {
+  letter-spacing: -1.4px;
+}
+
+@media screen and (min-width: 1024px) {
+  .hero__title {
+    letter-spacing: -4px;
+  }
+}
+
+.hero__description {
+  line-height: 26px;
+}
+
+.hero__form {
+  display: flex;
+}
+
+.hero__input {
+  flex-grow: 1;
+  padding: 0.75rem 1rem;
+  border-radius: 0;
+}
+
+.hero__content {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-evenly;
+}
+
+.hero__image {
+  max-width: 332px;
+  margin: auto;
+}
+
+@media screen and (min-width: 1024px) {
+  .hero__image {
+    max-width: 550px;
+  }
+}
+
+.hero__pay__methods {
```

Comment:
@FranzGB not a request but, sometimes a long file is better understood if you split it in tiny files based on the name, ex:
```
hero/
    |- image.css
    |- index.css
    |- container.css
    |- content.css
    |- input.css
    |- description.css
    |- .... and so on
```